src/js/view/controls/shortcuts-tooltip.ts

```diff
@@ -116,14 +116,18 @@ export default function (
     };
 
     const documentClickHandler = (e: Event) => {
-        const target = e.target as HTMLElement;
-        if (!/jw-shortcuts|jw-switch/.test(target.className)) {
+        const target: EventTarget | null = e.target;
+        if (target instanceof HTMLElement && !/jw-shortcuts|jw-switch/.test(target.className)) {
             close();
         }
     };
 
     const toggleClickHandler = (e: Event) => {
-        const toggle = e.currentTarget as HTMLElement;
+        if (!(e.currentTarget instanceof HTMLElement)) {
```

Comment:
I'd really rather not use runtime `instanceof ` checks. Why the change?

Reply:
as HTMLElement is basically saying "Ignore the check saying this isn't a typesafe operation, no matter what it's an HTMLElement". Since it's based on the target, we would need to check at runtime to know.

Reply:
given that we added the listener to an element, why do we need the check? We didn't need it before TypeScipt, do adding runtime checks to satisfy a build time library is a de-optimization.

Reply:
The casting is preventing a compile time error.

Reply:
You could also check `e.currentTarget.nodeType === 1`, but either way it doesn't relate directly to what this method is meant to do which is to change the `aria-checked` attribute value on the element you attached the listener to which is `shortcutToggleUi`, which should also be the value of `this`.

Reply:
I'm worried about the edge cases, especially in the case of documentClickHandler. `e.currentTarget` might be an element at the time that it's clicked, but removed from the DOM by the time the handler fires. That being said, I do understand not wanting to add work beyond the ticket scope so I've reverted the type-check enhancements.

Reply:
You can set attributes on an element no longer in the DOM. If the point it to modify the element the listener is attached to, then just use `this` instead of `event.currentTarget`.